src/Chapter1.hs

```diff
@@ -620,8 +626,14 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
 
+sumLast2 :: Int -> Int
+sumLast2 m = 
+  let n = abs m
+      lastD = lastDigit n
+      sLastD = div (mod n 100) 10
+      result = lastD + sLastD
```

Comment:
That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

One hint to make your solution even shorter: the standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that is actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(first, second) = divMod (mod x 100) 10
```

You can see how we could pattern match on the pair 🙂

Reply:
I don't know if I understand the pattern matching part, could you explain a bit more? But yeah divMod will simplify the code, I just didn't know that function existed.

Reply:
@fa7ad Don't worry about the pattern-matching too much for now! We explain it in detail in the next chapter. It is just an interesting fact, but maybe you can return back to this task later if you like :slightly_smiling_face: